### ISSUE TITLE
use too many draw calls, you should batch it. the fps has been downed.

### ISSUE THREAD

**Issue by ren19890419** [Arisego/UnrealLive2D#12]

**Comment by Arisego** [comment]
Yes, the render part of this plugin is not fully optimized. Some further work is needed on figuring out how those "draw-able" part of model works together. As sometimes model come with only one or two textures, there should be some way to just batch them together. 
But as our project decided not to use Live2D, I have little time on this plugin. I tried to upgrade it with UE4 and Cubism SDK, but got no chance to have a future study on optimizing. I might be back at some future moment, but it's not planned, I can not promise it.

**Comment by Jushua-bin** [comment]
i have the same problem and i try to batch the Vertex but is fail
